### PR TITLE
Moved the data table sort icons next to the column headings.

### PR DESCRIPTION
**Pull request by dsjen**
@rocha @clintonb 

FYI: @shnayder @marcotuts 

Before:
before-ff

After:
after-ff

### CODE REVIEW

**Comment by shnayder** [comment]
Arrow positions are definitely better.  Is it easy to also align the headers and the data?

File: analytics_dashboard/static/sass/_developer.scss [hunk 1]
```diff
@@ -229,6 +229,18 @@ table.dataTable thead th.sorting_desc:after {
   color: $edx-blue;
 }
 
+// position the arrows next to the header labels
+table.dataTable thead th.sorting:after,
```
Comment (clintonb) [inline analytics_dashboard/static/sass/_developer.scss]: Merge this with the block on line 220.

**Comment by clintonb** [comment]
Remove the duplicated code and :+1:

**Comment by dsjen** [comment]
@shnayder, here's a screenshot of the updated headers aligned with the text in the rows:
aligned-headers

@clintonb, css is consolidated.

**Comment by dsjen** [comment]
Also, I forgot to mention this was primarily to fix an issue in firefox where the sort weren't appearing.  All the screenshots are of the table in firefox too.

File: analytics_dashboard/static/sass/_developer.scss [hunk 2]
```diff
@@ -216,11 +216,22 @@ body.view-course-list {
...
+// Icon spacing and position the arrows next to the header labels
 table.dataTable thead th.sorting:after,
 table.dataTable thead th.sorting_asc:after,
 table.dataTable thead th.sorting_desc:after {
-  right: 3px;
+	position: relative;
+	top: 0px;
+	right: 0px;
+	display: inline;
+    padding-left: 8px;
```
Comment (clintonb) [inline analytics_dashboard/static/sass/_developer.scss]: Fix the spacing.

**Comment by clintonb** [comment]
:+1:

**Pull request merged**